api_callable_test.go

```diff
@@ -0,0 +1,85 @@
+package gax
+
+import (
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"testing"
+	"time"
+)
+
+var (
+	callSettings = &CallSettings{0, RetrySettings{[]string{"default"}, BackoffSettings{
+		100 * time.Millisecond,
```

Comment:
I think labeling the fields here (in all three structs) makes the code more readable)